src/diffpy/utils/diffraction_objects.py

```diff
@@ -507,6 +528,16 @@ def on_xtype(self, xtype):
             raise ValueError(_xtype_wmsg(xtype))
 
     def dump(self, filepath, xtype=None):
+        """Dump the xarray and yarray of the diffraction object to a two-column
+        file, with the associated information included in the header.
+
+        Parameters
+        ----------
+        filepath : str
```

Comment:
non programmers may not know that that the path includes the filename.  Best here is to give an example.

Reply:
@sbillinge added in the new commit. please review